EIPS/eip-4834.md

```diff
@@ -0,0 +1,221 @@
+---
+eip: 4834
+title: Hierarchical Domains Standard
+description: The Hierarchal Domains Standard is a standard for extremely generic name resolution.
+author: Pandapip1 (@Pandapip1)
+discussions-to: https://ethereum-magicians.org/t/erc-4834-hierarchical-domains-standard/8388
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-02-22
+requires: 165
+---
+
+## Abstract
+`ERC-4834` is a standard for generic name resolution with access control. It permits contracts that implement `ERC-4834` ("domains") to be addressable with a more human-friendly name, similar to `ERC-137`.
```

Comment:
Oh, sorry, I forgot to mention that you have to link to an EIP the first time you mention it.

Reply:
Done